contracts/HMToken.sol

```diff
@@ -153,7 +151,7 @@ contract HMToken is HMTokenInterface {
         if (_to == address(this)) return false; // Preclude sending tokens to the contract.
         if (balances[msg.sender] < _value) return false;
 
-        balances[msg.sender] = balances[msg.sender].sub(_value);
+        balances[msg.sender] = balances[msg.sender] - _value;
```

Comment:
Is the `-` better than the safemath version?

Reply:
It's better from a clarification perspective, meaning: you check for underflows in the above `if`, so it's useless to use that function from the SafeMath library (I think it's even a bit more gas-costly? not sure, though). Either I dropped the `if` (should I do that?), either I obeyed the recommendation from the certiK report (drop the `sub`).

Reply:
@posix4e might have an opinion but seems reasonable to me